addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -388,7 +388,6 @@ registerModel({
          * @param {number} settings.voice_active_duration
          * @param {boolean} [settings.is_discuss_sidebar_category_channel_open]
          * @param {boolean} [settings.is_discuss_sidebar_category_chat_open]
-         * @param {Object} [payload.volume_settings]
```

Comment:
1st commit: cleaning of docstring is unrelated to this commit, which is rename. Could be in another "cleaning" commit